### PR TITLE
Feat(content): Incipias Tractor Beam

### PR DESCRIPTION
**Pull request by RisingLeaf**
## Summary
As discussed previously the Incipias are getting their own tractor beam that i more advanced than the human version.
I gave it to their one mining ship with turret mount.

- [x] todo make description more awesome

Idea for the future:
sing fleet formations you could make a group where some mine asteroids and some collect, I imagine one gun ship and one or two small freighters with tractor beams.
would need a new freighter with turret mount though

## Testing Done
Played around a bit in the game, watched their ships etc.
balancing probably needs a look.

## Save File
Just go to their system.

## Artwork Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I created a PR to the [endless-sky-assets repo](https://github.com/endless-sky/endless-sky-assets/pull/175)
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi/pull/418)

### CODE REVIEW

**Review by Quantumshark — COMMENTED** [review]
For comparison, the Human tractor beam takes 22 outfit space, has 300 range, 7 `tractor beam`, and has 2.1 firing energy and .8 firing heat.
So, this is longer-range, stronger, and slightly more efficient, but slightly larger - i.e, much better overall.

File: data/incipias/incipias outfits.txt [hunk 1]
```diff
@@ -180,6 +179,33 @@ outfit "Star Tail Storage"
...
+		"hardpoint offset" 0.
+		sound "tractor beam"
+		"hit effect" "horizon grappler"
+		"tractor beam" 9
+		"velocity" 400
+		"lifetime" 1
+		"reload" 1
+		"firing energy" 1.9
+		"firing heat" .6
+	description `The pinacle of Incipian technology sees itself represented in this mining utility, perfected over centuries this tractor beam is far beyond anything they have developed so far. Its name originates from the first prototypes that pulled asteroids from the "horizon".`
```
Comment (tibetiroka) [inline data/incipias/incipias outfits.txt:201]:
```suggestion
	description `The pinnacle of Incipian technology sees itself represented in this mining utility. Perfected over centuries, this tractor beam is far beyond anything they have developed so far. Its name originates from the first prototypes that pulled asteroids from the "horizon".`
```

**Comment by RisingLeaf** [comment]
> For comparison, the Human tractor beam takes 22 outfit space, has 300 range, 7 `tractor beam`, and has 2.1 firing energy and .8 firing heat. So, this is longer-range, stronger, and slightly more efficient, but slightly larger - i.e, much better overall.

It is supposed to be a better version, question being, how much would be in bounds, would it be more ok if I increase energy and heat consumption?

**Comment by Quantumshark** [comment]
I'd say put the energy up a fair bit, maybe to 3.2, as a cost to the higher range and `tractor beam` power. You could up the heat from .6 to 1.1 in a similar vein, but that's not as significant from the balance perspective either way, as a given amount of heat is much easier to deal with than the same amount of energy.

**Comment by RisingLeaf** [comment]
I adjusted the outfit

**Review by roadrunner56 — COMMENTED** [review]
I took some liberties with the description; up to you whether you want to include it or not.

**Review by roadrunner56 — APPROVED** [review]

**Review by Quantumshark — APPROVED** [review]

**Pull request merged**